proofread releases (#2618)
centreon/centreon-documentation: versioned_docs/version-23.10/releases/lifecycle.md

### From version 21.10

> From version 21.10, versions have been supported for two years. The lifecycle of a version is divided into two phases of 12 months each.

Reviewer: ?

> From version 21.10, versions have been supported for two years. The lifecycle of a version is divided into two phases of 12 months each.

Reviewer: ```suggestion
Since version 21.10, versions are supported for two years. The lifecycle of a version is divided into two phases of 12 months each.
```
